src/metabase/fingerprinting/comparison.clj

```diff
@@ -0,0 +1,117 @@
+(ns metabase.fingerprinting.comparison
```

Comment:
same thing I think we should change all the `metabase.fingerprinting` namespaces to `metabase.xray` or something